ngrok-java/src/main/java/com/ngrok/EndpointBuilder.java

```diff
@@ -58,7 +66,19 @@ public T proxyProto(ProxyProto proxyProto) {
     }
 
     /**
-     * Returns a list of strings representing allowed CIDR addresses for this endpoint.
+     * Sets the policy for this endpoint.
+     *
+     * @param policy the policy for the builder
+     * @return An instance the builder represented by type T
+     */
+    public T policy(final String policy) {
```

Comment:
`String` here is kind of unfortunate (as opposed to a proper Policy object) 🤷

Reply:
Agree but for V1 it seems like this is what we're doing in all the SDKs before adding full builder pattern. :(